Makefile

```diff
@@ -184,6 +184,19 @@ docker-build-storageInitializer:
 docker-push-storageInitializer: docker-build-storageInitializer
 	docker push ${KO_DOCKER_REPO}/${STORAGE_INIT_IMG}
 
+kubebuilder:
```

Comment:
Why do we need kb as a dependency? This is for generating controllers, right?

Reply:
this is for `make test` which requires local apiserver/etcd which is installed part of kubebuilder